staticaddr/sql_store.go

```diff
@@ -0,0 +1,194 @@
+package staticaddr
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/jackc/pgx/v4"
+	"github.com/lightninglabs/loop/loopdb"
+	"github.com/lightninglabs/loop/loopdb/sqlc"
+	"github.com/lightningnetwork/lnd/keychain"
+)
+
+// SqlStore is the backing store for static addresses.
+type SqlStore struct {
+	baseDB  *loopdb.BaseDB
+	nowTime func() time.Time
```

Comment:
Or alternatively we could use LND's `clock.Clock` (which also has a nice test version `clock.TestClock`).